### ISSUE TITLE
ui, wallet, introducer, and harvester port conflicts.  same ssl cert as flax.

### ISSUE THREAD

**Issue by ddowhy** [DogeChia/doge-chia#18]
**Describe the bug**
dogechia uses the same ui, wallet, and introducer ports as flax.  it also uses the same SSL cert as flax. these conflicts cause doge-chia apps to fail.  seeing that you forked from flax (and not vise-versa), you should change these.  

it also uses the same harvester port as avocado, but I think that one came later.  that will still cause proofs not to be found if both harvesters are running on the same machine.

**To Reproduce**
you can see the port conflicts easily yourselves in the config files.
you can also diff the ca cert with flax to see they are the same.   

**Expected behavior**
no port or ssl conflicts

**Comment by Ilmairinen** [comment]
You're right and we're working on a fix. To do this non-disruptively is a challenge. Thank you for reporting this issue. I'll let you know when an early preview of the next build is available, should you like to try it.

**Comment by Ilmairinen** [comment]
Port conflicts are now resolved in 1.0.14 release. SSL certificate change will be addressed in an upcoming release.

**Comment by Ilmairinen** [comment]
What's new in 1.0.15

Fixed Port Conflicts:

Changed DogeChia wallet port from 6883 to 16823 to fix port conflict with Flax - sorry about that Flax.

Won't fix:

    Avocado blockchain Harvester port 6869 conflicts with DogeChia Harvester port 6869.

    Thyme blockchain rpc_port 16759 conflicts with DogeChia rpc_port 16759.

    Thyme blockchain Timelord port conflicts with DogeChia Timelord port 16886.

    Lucky wallet port 16669 conflicts with DogeChia timelord_launcher port 16669

DogeChia is older than Avocado, Thyme and Lucky.
DogeChia did not create these port conflicts and we will not change our code to fix them.
